src/components/PrizePage/Prizes.js

```diff
@@ -40,6 +125,41 @@ export default function Prizes() {
 		prizeImageMap.set(id, childImageSharp.fixed);
 	}
 	return (
-		<ComingSoon alignment='center'/>
+		<>
+			<Container maxWidth='md'>
+				<Grid container spacing={3} className={classes.prizeGrid}>
+					{allPrizes.map(prize =>
+						<Grid item xs={12} sm={6} md={4} key={prize.track}>
+							<Box display='flex' justifyContent='center' flexDirection='column'>
+								<Box className={classes.imageBox}>
+									<Img fluid={iconImageMap.get(prize.img)}
+										style={{ height: '70%', width: '70%' }}
+										objectFit = "contain"
+									/>
+								</Box>
+								<Box className={classes.prizeCard}>
+									<Typography variant='h6' component='h3'>
+										{prize.track}
+									</Typography>
+									<Box paddingX='20%'>
+										<Squiggle />
+									</Box>
+									<Box className={classes.prizeDescription}>
+										<Typography style={{ color: '#a1a1a1',
+											fontSize: '14px',
+											textTransform: 'uppercase' }}>
+											prize
+										</Typography>
+										<Typography style={{ paddingLeft: '10px', paddingRight: '10px' }}>
```

Comment:
Can possibly move this into the `makeStyles`, which will enable us to use `theme.spacing`